Dockerfile

```diff
@@ -39,8 +39,8 @@ RUN \
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash && \
 export NVM_DIR="$HOME/.nvm" && \
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" && \
-nvm install 16.20.0
-ENV VSCODE_NODEJS_RUNTIME_DIR="$HOME/.nvm/versions/node/v16.20.0/bin/"
+nvm install 18.15.0
+ENV VSCODE_NODEJS_RUNTIME_DIR="$HOME/.nvm/versions/node/v18.15.0/bin/"
```

Comment:
@svor have you tried without nodejs. I mean, we used to require nodejs 16 because on UBI9 based image because of VS Code required libcrypto v1 but now that VS Code has been updated to include nodejs 18...

```suggestion
```

Reply:
@l0rd in ansible-creator-ee image I see the error about libcrypto when using builtin node:
```
$ /checode/checode-linux-libc/node --version
/checode/checode-linux-libc/node: error while loading shared libraries: libcrypto.so.1.1: cannot open shared object file: No such file or directory
```
![screenshot-devspaces apps sandbox-stage gb17 p1 openshiftapps com-2023 10 04-13_46_09](https://github.com/devspaces-samples/ansible-devspaces-demo/assets/1271546/69b618e9-1b4a-422d-a454-43a22d5bbc67)

Reply:
`registry.access.redhat.com/ubi8/nodejs-18:1-71` is used to prepare VS Code assembly.
It uses Node.js that depends on the `libcrypto.so.1.1`.
There is an error if the required library is absent in the container where the VS Code is running.

<img width="830" alt="Screenshot 2023-10-04 at 14 02 33" src="https://github.com/devspaces-samples/ansible-devspaces-demo/assets/5676062/7ccf30ed-4857-42b3-8aa6-693917053f08">

Reply:
@l0rd 
so, it looks like https://github.com/eclipse/che/issues/21778 is still actual

Reply:
Ok thanks for looking a it. @svor you can ignore my comment then.

Reply:
since the latest in UBI8 is 18.17.1, why not use something even newer?

Reply:
looking at [prerequisites](https://github.com/microsoft/vscode/wiki/How-to-Contribute#prerequisites), the node version should be >=18.15.x and <19
so, updating to 18.18.0